src/prog_models/linear_model.py

```diff
@@ -28,6 +29,53 @@ class LinearModel(PrognosticsModel, ABC):
         * events:  list[str] - event keys
     """
 
+    def __init__(self):
+        super().__init__()
+
+        """
```

Comment:
Use a regular comment here instead of docstring. Docstrings are used as labels at the beginning of a file/class/function